roles/edpm_nova/tasks/configure.yml

```diff
@@ -24,8 +24,8 @@
     path: "{{ item.path }}"
     setype: "container_file_t"
     state: "directory"
-    owner: "{{ item.owner | default(omit) }}"
-    group: "{{ item.group | default(omit) }}"
+    owner: "{{ item.owner | default(ansible_user) }}"
+    group: "{{ item.group | default(ansible_user) }}"
```

Comment:
default omit was used so that we did not modify the permission of the diretores unless we specified them

we could make this ansible_user but it should not be required.

if we want to proceed with this then /var/lib/nova, /var/lib/nova/instances and  "/var/log/containers/nova", should probably be set to the nova user

Reply:
what do you think is the best way to handle https://github.com/openstack-k8s-operators/data-plane-adoption/pull/219#discussion_r1446963270 ?